book.rb

```diff
@@ -0,0 +1,18 @@
+# Create a class Book with the following:
+# @title and @author instance variables, should be initialized in the constructor.
+# Setters and getters for instance variables (remember about attr_accessor).
+
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(person, date)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] In the Person class you need to initialize the `rentals` array as you have done here.

- [ ] In the Person class you need to add a similar method to this `add_rental` that takes in `book` and `date` and instantiates a rental

  ```
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ```